### ISSUE TITLE
Township in search forms is not persisted

### ISSUE THREAD

**Issue by jusabatier** [georchestra/mapstore2-cadastrapp#155]
Currently, when we close cadastrapp addon and reopen it, it doesn't persit last used township for search.

In the mfapp addon, we have the ability to fill the township the first time, and it's reused in all forms after.
It's an important feature as a lot of our users work in a city hall and request always the same township, so they don't want to fill it for each search.

@MaelREBOUX @landryb @catmorales

**Comment by landryb** [comment]
is that something to keep in localstorage ?

**Comment by jusabatier** [comment]
I think that's the way it's handled in mfapp addon.

And the value is replaced with the one of the last search.

**Comment by MaelREBOUX** [comment]
I totally agree : we need to use the local storage to memorize the current / last municipality.
